Makefile

```diff
@@ -356,29 +356,14 @@ convert-release-notes:
 		pandoc --from=markdown --to=rst $${file} > $${file%%.md}.rst; \
 		done
 
-list-authors:
```

Comment:
This target was half-baked, and was already getting out of sync with the other two author-related targets.